pkgs/development/tools/gen-license/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchCrate
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gen-license";
+  version = "0.1.2";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-YZcycLQ436cjr2YTT7TEyMdeLTOl9oEfa5x3lgnnYyo=";
+  };
+
+  cargoLock.lockFile = "${src}/Cargo.lock";
+
+  buildInputs = lib.optional stdenv.isDarwin [ Security ];
```

```suggestion
  buildInputs = lib.optional stdenv.isDarwin Security;
```
oops! this is my bad (causes nested lists without this change)